src/cmd-clean

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+set -xeuo pipefail
+
+dn=$(dirname $0)
+. ${dn}/cmdlib.sh
+
+# This has some useful sanity checks
+prepare_build
```

Comment:
seems a bit hacky, i guess all of this will be redone at some point in another language so +1